interface/interface.dm

```diff
@@ -53,6 +53,41 @@
 	src << link("https://github.com/ColonialMarines-Mirror/ColonialMarines-2018")
 	return
 
+/client/verb/webmap()
+	var/ship_link
+	var/ground_link
+	set name = "webmap"
+	set desc = "Opens the webmap"
+	set hidden = 1
+	var/choice = alert("Do you want to view the ground or the ship?",,"Ship","Ground","Cancel")
+	if(choice == "Cancel")
```

Comment:
Add a `switch` to these options and it's good.
You can run the "Cancel" simply as an `else` at the end.